### PR TITLE
Package security updates

### PR DESCRIPTION
**Pull request by yjpa7145**

### CODE REVIEW

File: tests/test.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-import os from 'os';
-import fs from 'fs-extra';
-import path from 'path';
-import test from 'ava';
-import nock from 'nock';
-import sinon from 'sinon';
-import AWS from 'aws-sdk';
-import archiver from 'archiver';
-import { runTask, runServiceFromActivity } from '../index';
+const os = require('os');
```
Comment (markdboyd) [inline tests/test.js:3]: It doesn't matter, but why the change?

**Review by markdboyd — APPROVED** [review]

File: tests/test.js (hunk 1 above)
Comment (yjpa7145) [reply tests/test.js:3]: Honestly, I'm not sure how this was working in the first place. When I upgraded to newer package versions, these tests started failing.
